scoap3/tasks.py

```diff
@@ -76,11 +77,11 @@ def _create_article(data, licenses):
         "abstract": data["abstracts"][0].get("value", ""),
     }
     article, _ = Article.objects.get_or_create(**article_data)
-    print(data.get("_created"))
     article._created_at = data.get("_created")
     article._updated_at = data.get("_updated")
     article.related_licenses.set(licenses)
     article.save()
+    time.sleep(1)
```

Comment:
Wait for one sec until the index will be refreshed and data will be available on OpenSearch. By default, the index is refreshed every 1 sec. https://opensearch.org/docs/latest/api-reference/index-apis/create-index/#dynamic-index-settings
I don't think it is the most beautiful solution. 
We can also just disable the automatic refresh and manually refresh it when we needed. It means, just when we create/delete/edit something.